packages/core/src/integrations/inboundfilters.ts

```diff
@@ -84,8 +64,7 @@ export function _mergeOptions(
   };
 }
 
-/** JSDoc */
-export function _shouldDropEvent(event: Event, options: Partial<InboundFiltersOptions>): boolean {
+function _shouldDropEvent(event: Event, options: Partial<InboundFiltersOptions>): boolean {
```

Comment:
Why does this function start with underscore?

Reply:
they are "private" - we are not super consistent there, sometimes we prefix them and sometimes not. but specifically here I did not change that, just removed the `export` as we don't actually use it anywhere!